plutus-tx/tutorial/tutorial-doctests.hs

```diff
@@ -1,2 +1,2 @@
 import           Test.DocTest
-main = doctest ["-pgmL markdown-unlit", "-isrc", "-itutorial", "tutorial/Tutorial.lhs"]
+main = doctest ["-pgmL markdown-unlit", "-isrc", "-itutorial", "--fast", "-XTemplateHaskell", "-XScopedTypeVariables", "tutorial/Tutorial.lhs"]
```

Comment:
I don't think this will run the doctests on the prelude module, I think you also have to pass that at the end?

Reply:
I didn't expect that either, but they do. That's why I had to add the extensions (and the tutorial doctest fails if you change the outputs in the Prelude)